tsconfig.json

```diff
@@ -28,6 +28,7 @@
     "skipLibCheck": true,
     "paths": {
       "@dcos/http-service": ["src/typings/dcos__http-service"],
+      "@extension-kid/data-layer": ["packages/@extension-kid/data-layer"],
```

Comment:
should we add `notification-service` and `toast-notifications` to this config too? We can do that in another PR I just didn't know about this list.

Reply:
If they don't need to be required with types we can leave them of, but we could also add them for the sake of completeness :)